modules/electromagnetics/include/kernels/VectorSecondTimeDerivative.h

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "VectorTimeKernel.h"
+
+/**
+ *  The second time derivative operator for vector variables
+ */
+class VectorSecondTimeDerivative : public VectorTimeKernel
```

Comment:
this could go in framework/
your call

Reply:
I've gone back and forth about it. Will consider moving.

Reply:
As this is only really relevant in the context of the EM module for now, I will leave it here.